guides/doc-Release_notes/master.adoc

```diff
@@ -0,0 +1,27 @@
+include::common/attributes.adoc[]
+
+:numbered:
+:toc:
+:toc-placement: left
+
+ifdef::katello[]
+= Foreman {ProjectVersion} and Katello {KatelloVersion} Release Notes
+endif::[]
+
```

Comment:
If it's "foreman", then there'd be no headline if I read this correctly. I think you need to add the following:

```suggestion
ifndef::katello[]
= Foreman {ProjectVersion} Release Notes
endif::[]
```